dom.bs

```diff
@@ -6738,6 +6741,11 @@ invoked, must run these steps:
  <li><p>Set <var>shadow</var>'s <a for=ShadowRoot>delegates focus</a> to <var>init</var>'s
  {{ShadowRootInit/delegatesFocus}}.
 
+ <li><p>If <a>this</a>'s <a for=Element>custom element state</a> is "<code>precustomized</code>"
+ or "<code>custom</code>", set <var>shadow</var>'s
+ <a for=ShadowRoot>available to element internals</a> property to true. Otherwise, set it to
+ false.
```

Comment:
```suggestion
 <li><p>If <a>this</a>'s <a for=Element>custom element state</a> is "<code>precustomized</code>" or 
 "<code>custom</code>", then set <var>shadow</var>'s 
 <a for=ShadowRoot>available to element internals</a> to true.
```
It seems to me we don't have to set it false as that's the default, right?

Reply:
You're right.